raiden/tests/integration/api/test_restapi.py

```diff
@@ -1197,3 +1197,65 @@ def test_api_deposit_limit(
     assert_proper_response(response, HTTPStatus.CONFLICT)
     response = response.json()
     assert response['errors'] == 'The deposit of 10001 is bigger than the current limit of 10000'
+
+
+@pytest.mark.parametrize('number_of_nodes', [2])
+def test_channel_history_events(api_backend, raiden_network, token_addresses):
+    _, app1 = raiden_network
+    amount = 200
+    identifier = 42
+    token_address = token_addresses[0]
+    target_address = app1.raiden.address
+
+    api_server, _ = api_backend
+    our_address = api_server.rest_api.raiden_api.address
+
+    transfer = {
+        'initiator_address': to_checksum_address(our_address),
+        'target_address': to_checksum_address(target_address),
+        'token_address': to_checksum_address(token_address),
+        'amount': amount,
+        'identifier': identifier,
+    }
+
+    request = grequests.post(
+        api_url_for(
+            api_backend,
+            'transfertotargetresource',
+            token_address=to_checksum_address(token_address),
+            target_address=to_checksum_address(target_address),
+        ),
+        json={'amount': amount, 'identifier': identifier},
+    )
+    response = request.send().response
+    assert_proper_response(response)
+    response = response.json()
+    assert response == transfer
+
+    # test endpoint with partner
```

Comment:
Right now there is only one transfer, so we're not testing if filtering stuff out actually works. Please add a second transfer to a different address and test that as well.

Reply:
Nice catch. I'll do it. Thanks.